pkg/query-service/telemetry/telemetry.go

```diff
@@ -213,8 +206,11 @@ func createTelemetry() {
 
 	telemetry.SetTelemetryEnabled(constants.IsTelemetryEnabled())
 
-	ticker := time.NewTicker(HEART_BEAT_DURATION)
-	activeUserTicker := time.NewTicker(ACTIVE_USER_DURATION)
+	// Create a new scheduler
+	s := gocron.NewScheduler(time.UTC)
```

Comment:
Context usage doesn't look right.

Please refer to [this](https://github.com/SigNoz/saas-scripts/blob/main/sqlite-backup/cron.go#L14).

Reply:
I am using V1 and the above reference uses v2.